ta/pkcs11/src/processing_aes.c

```diff
@@ -14,6 +15,385 @@
 #include "processing.h"
 #include "serializer.h"
 
+/*
+ * Authenticated ciphering: (CCM / GCM)
+ *
+ * As per PKCS#11, CCM/GCM decrypt shall not revealed the data until the
+ * decryption is completed and the mac verified. The SKS TA must retain the
+ * ciphered data until the CCM finalization. To do so, arrays of decrypted
+ * data are allocated during AE update processing and copied into client
+ * buffer at AE finalization.
+ *
+ * As per PKCS#11, CCM/GCM decrypt expect the tag/mac data to be provided
+ * inside the input data for DecryptUpdate() and friends. But the DecryptFinal
```

Comment:
s/DecryptUpdate/C_DecryptUpdate/
s/DecryptFinal/C_DecryptFinal/

above and below (few occurrences)